tests/php/Forms/TreeMultiselectFieldTest.php

```diff
@@ -10,22 +10,57 @@ class TreeMultiselectFieldTest extends SapphireTest
 {
     protected static $fixture_file = 'TreeDropdownFieldTest.yml';
 
+    public function testEmptyChoiceReadonly()
+    {
+        $field = new TreeMultiselectField('TestTree', 'Test tree', File::class);
+
+        $asdf = $this->objFromFixture(File::class, 'asdf');
+        $subfolderfile1 = $this->objFromFixture(File::class, 'subfolderfile1');
+
+        $schemaStateDefaults = $field->getSchemaStateDefaults();
+        $this->assertArraySubset(['id' => 'TestTree', 'name' => 'TestTree', 'value' => 'unchanged'], $schemaStateDefaults, $strict = true);
+
+        $items = $field->getItems();
+        $this->assertCount(0, $items, $message = 'there must be no items selected');
+
+        $html = $field->performReadonlyTransformation()->Field();
+
+        $this->assertEquals(
+            <<<"HTML"
```

Comment:
Minor: The double-quotes here are unnecessary (same as no quotes) - Perhaps you meant for a nowdoc, but that's specifically single-quotes.

Reply:
well, not really meant anything, just copy pasted this bit from the original code :)
I'm rewriting this test to be more unit, so will remove the html block.